prov/verbs/src/verbs_domain.c

```diff
@@ -521,18 +523,21 @@ int vrb_fabric(struct fi_fabric_attr *attr, struct fid_fabric **fabric,
 	if (!fab)
 		return -FI_ENOMEM;
 
+	ofi_mutex_lock(&vrb_info_mutex);
```

Comment:
This assignment ```*info = vrb_util_prov.info;``` was made in the variable declaration section outside of holding the mutex.  It needs to move inside the lock now.